src/app/plan/components/Wizard/HooksStep.tsx

```diff
@@ -0,0 +1,220 @@
+      return {
+        cells: [migHook.hookName, migHook.image, migHook.clusterTypeText, migHook.phase],
+      }
+    });
+    actions = [
+      {
+        title: 'Edit',
+        onClick: (event, rowId, rowData, extra) => {
+          const currentHook = migHookList.find((hook) => hook.hookName === rowData.name.title)
+          setInitialHookValues(currentHook)
+          setIsAddHooksOpen(true)
+          watchHookAddEditStatus(rowData.name.title);
+
+        }
+      },
+      {
+        title: 'Delete',
+        onClick: (event, rowId, rowData, extra) => {
+          removeHookRequest(rowData.name.title, rowData["migration-step"].title)
+        }
+      }
+    ]
+
+  }
+  if (rows.length === 0) {
+    rows = [{
+      heightAuto: true,
+      cells: [
+        {
+          props: { colSpan: 8 },
+          title: (
+            <Bullseye>
+              <EmptyState variant={EmptyStateVariant.full}>
+                <EmptyStateIcon icon={PlusCircleIcon} />
+                <Title headingLevel="h4" size="lg">
+                  No hooks have been added to this migration plan.
+                </Title>
+              </EmptyState>
+            </Bullseye >
+          )
+        },
+      ]
+    }]
+  }
+
+  const hooksFormContainerStyles = classNames(spacing.mtSm, styles.formContainerStyles)
+  return (
+    <Grid>
+      <GridItem className={spacing.mtSm}>
+        <TextContent>
+          <Text component={TextVariants.h2}>
+            Hooks
+          </Text>
+        </TextContent>
+      </GridItem>
+      <GridItem>
+        <TextContent>
+          <Text component={TextVariants.p}>
+            Hooks are commands that can be run at various steps in the migration process. They are defined in a container image or an Ansible playbook and can be run on either the source or target cluster.
+          </Text>
+        </TextContent>
+      </GridItem>
+      {isAddHooksOpen &&
+        <GridItem className={hooksFormContainerStyles}>
+          <HooksFormContainer
+            onAddEditHookSubmit={onAddEditHookSubmit}
+            initialHookValues={initialHookValues}
+            setInitialHookValues={setInitialHookValues}
+            setIsAddHooksOpen={setIsAddHooksOpen}
+            isAddHooksOpen={isAddHooksOpen}
+            isEditHook={false}
+            {...props} />
+        </GridItem>
+      }
+      {(!isAddHooksOpen) &&
+        <React.Fragment>
+          {isFetchingHookList ?
+            <Bullseye>
+              <EmptyState variant="large">
+                <div className="pf-c-empty-state__icon">
+                  <Spinner size="xl" />
+                </div>
+                <Title headingLevel="h2" size="xl">
+                  Finding hooks for this migration plan...
+                </Title>
+              </EmptyState>
+            </Bullseye>
+            :
+            <GridItem className={hooksFormContainerStyles}>
+              <Grid gutter="md">
+                <GridItem span={4}>
+                  <Button
```

Comment:
In the mocks it looks like there is a search filter to the left of the "Add hook" button. @vconzola did we want filters, sorting, and pagination in this table? @ibolton336 we can do that in a followup PR, but let me know if you want me to explain the `useFilterState` / `useSortState` / `usePaginationState` stuff.

Reply:
Think this is a good candidate for an issue and a follow up PR.

Reply:
Sure that would be great to have you walk me through that. Going to leave the filter out for this pass.